services/utils/config.py

```
@@ -0,0 +1,43 @@
+    API_PORT: int = 8000
+    MODEL_PATH: str = "./models/trained"
+    PROMPT_INJECTION_THRESHOLD: float = 0.75
+    SHUTDOWN_TIMEOUT: float = 10.0
+    CORS_ALLOWED_ORIGINS: str = "https://localhost:3000,https://localhost:5173"
+    CORS_ORIGINS: str = "http://localhost:3000"
+    
+    CB_FAILURE_THRESHOLD: int = 3
+    CB_RECOVERY_TIMEOUT_S: float = 30.0
+    CB_HALF_OPEN_MAX_CALLS: int = 1
+    
+    AUDIT_HMAC_SECRET: str = "tenet-audit-dev-secret"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "AUDIT_HMAC_SECRET"

(S105)